src/controllers/organizations.js

```diff
@@ -0,0 +1,146 @@
+const { default: mongoose } = require('mongoose');
+const { Organization, User } = require('../models/user');
+
+async function getOneOrganization(req) {
+    const { id: orgaId } = req.params;
+    const requiredUserFields = [
+        'id',
+        'firstName',
+        'lastName',
+        'email',
+        'profileImage',
+    ];
+
+    const orgaExecludedFields = {
+        email: 0,
```

Comment:
@muhammed-shihebi if we hide the email of an organization. Isn't it logical that we also hide it from the user? or we expose email for both of them in public profiles. what you think? so i can implement that in my side too!